third-party/highlight.js

```diff
@@ -0,0 +1,32 @@
+import highlightjs from 'highlight.js/lib/highlight';
+import 'highlight.js/styles/github.css';
+
+import bash from 'highlight.js/lib/languages/bash';
```

Comment:
Using all of the languages that are defined in the old `highlight.pack.js` file. We might want to standardize this set of languages so that `dash-table` and `dash-core-components` support the same languages.